components/omega/doc/design/PGrad.md

```diff
@@ -0,0 +1,224 @@
+(omega-design-pressure-grad)=
+# Pressure Gradient
+
+
+## 1 Overview
+The pressure gradient will be responsible for computing the horizontal gradients of both the pressure and geopotential terms for the non-Boussinesq primitive equations implemented in Omega.
+In the non-Boussinesq model, the conserved quantity is mass rather than volume.
+In Omega the prognostic variable $\tilde{h}$ is a pseudo thickness, rather than geometric thickness in m as in a Boussinesq model.
```

Comment:
But isn't the pseudo thickness also in m (i.e. units of length)? If so, it's confusing to mention "in m" for one but not the other.